packages/eslint-plugin-internal/src/rules/plugin-test-formatting.ts

```diff
@@ -478,33 +491,99 @@ export default createRule<Options, MessageIds>({
       }
     }
 
-    const invalidTestsSelectorPath = [
-      AST_NODE_TYPES.CallExpression,
-      AST_NODE_TYPES.ObjectExpression,
-      'Property[key.name = "invalid"]',
-      AST_NODE_TYPES.ArrayExpression,
-      AST_NODE_TYPES.ObjectExpression,
-    ];
-
     return {
       // valid
       'CallExpression > ObjectExpression > Property[key.name = "valid"] > ArrayExpression':
         checkValidTest,
       // invalid - errors
-      [invalidTestsSelectorPath.join(' > ')]: checkInvalidTest,
-      // invalid - suggestions
       [[
-        ...invalidTestsSelectorPath,
-        'Property[key.name = "errors"]',
-        AST_NODE_TYPES.ArrayExpression,
+        AST_NODE_TYPES.CallExpression,
         AST_NODE_TYPES.ObjectExpression,
-        'Property[key.name = "suggestions"]',
```

Comment:
in #5379 we stopped checking output formatting.
suggestions only ever have output code, so no sense even bothering to check them!